backend/Bindings/Edna.Bindings.User/Models/User.cs

```diff
@@ -0,0 +1,15 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Edna.Bindings.User.Models
+{
+    public class User
+    {
+        public string FamilyName { get; set; }
+        public string GivenName { get; set; }
+        public string Email { get; set; }
+        public string Role { get; set; }
```

Comment:
I created this file on same lines as the MemberDto from Edna.User

Reply:
Keeping an `enum` should not cause much trouble anyways since they can easily map to `string`. However, moving to them has a benefit of not running into issues where we mistyped something to not be `teacher | student`. I'm totally fine deferring this work to another PR though.

Reply:
we can make this change separately.